lib/l10n/arb/app_en.arb

```diff
@@ -0,0 +1,82 @@
+{
+"language":"English",
+"@language":{
+  "description":"The current Language"
+},
+
+"closeIcon":"close icon",
+"searchIcon":"search icon",
```

Comment:
remove this as well as we removed`searchIcon` from `ValueKey`

Reply:
Have added indentation and removed unused keys and made a commit for this file. You can check

Reply:
You can comment if you need any more modifications.